src/data/tasks/personal-details.json

```diff
@@ -0,0 +1,100 @@
+{
+  "id": "personal-details",
+  "points": 2,
+  "icon": "Face",
+
+  "eng": {
+    "title": "Personal Details",
+    "description": "Earn Khetha points by telling us a bit about yourself and how you would like to receive your airtime rewards",
+    "questions": [
+      {
+        "title": "Do you give us permission to save your information?",
+        "description": "Your information will be kept confidential",
+        "format": "buttons",
+        "options": [
+          "Yes",
+          "No"
+        ]
+      },
+
+      {
+        "title": "What will your gaming name be?",
+        "description": "You can choose an alias if you want",
+        "format": "string"
+      },
+
+
+      {
+        "title": "Which network provider are you with?",
+        "description": "This information will be used for airtime prizes",
+        "format": "checkboxes",
+        "options": [
+          "Cell C",
+          "MTN",
+          "rain",
+          "Telkom (8ta)",
+          "Virgin Mobile",
+          "Vodacom"
+        ]
+      },
+
+      {
+        "title": "What do you prefer as a reward for playing?",
+        "format": "buttons",
+        "options": [
+          "Airtime",
+          "Data"
+        ]
+      },
+
+      {
+        "title": "What is your year of birth?",
+        "format": "date"
```

Comment:
This has been changed in the sheet just require the year… is there a way we can accommodate this?

Reply:
(We can probably get feedback on this from SSA, as part of sharin g the preview.)